script/entrypoint

```diff
@@ -19,7 +19,8 @@ fi
 
 if [ "$#" = 0 ]; then
     # Default to 20 workers because most of the tests are io-bound
-    exec pytest --black --flake8 --mypy-ignore-missing-imports --pydocstyle --numprocesses 20
```

Comment:
This was causing test failures because of mypy or pytest-mypy, which is due to the free floating `data.py` files in each of the sql tests being treated as a duplicate module (https://app.circleci.com/pipelines/github/mozilla/bigquery-etl/6996/workflows/ad741abd-36a0-4649-bf39-1e8e0d1616b1/jobs/35787).

Reply:
I think it has to do with pytest collecting all of the python files under the test directory as individual modules (although this is mostly conjecture). Since I have a `data.py` file in each test, these appear to be the same module several times. When run outside of pytest, there is no `__init__.py` file and therefore it's ignored. 

If I add a `__init__.py` file to each test going up the directory levels so each test appears like a unique module, then the pytest mypy plugin will work correctly. I want these scripts to be standalone, not part of a module though.